packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -451,7 +451,7 @@ export class GutenbergEditorPage {
 		const frame = await this.getEditorFrame();
 
 		await Promise.all( [
-			this.page.waitForNavigation( { url } ),
+			this.page.waitForNavigation( { url: url, waitUntil: 'domcontentloaded' } ),
```

Comment:
It would appear that sometimes, waiting until the published post page fires `load` causes the tests to time out.

Ideally, the cause for the excessively long `load` event is fixed, though I suspect it has to do with occasional network and/or backend hiccups.

Reply:
I could be wrong, but I felt like I did a really long debug of this once, and I believe the culprit was ads! Since these are free sites, we show ads on them, and I think those can on occasion get really bogged down and delay certain page events (even though the page can be interacted with)

Reply:
I totally forgot about ads on free sites! That would explain why sometimes the loading takes much longer than expected.

`domcontentloaded` should allow us an easy way out of relying on slow third party ad services. If required, we can also use `page.route` to deny ads from being served, but let's do that if this approach begins to fail.